### PR TITLE
PR to ensure that pipeline works

### PR DESCRIPTION
**Pull request by ttstanley**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,182 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger:
+  branches:
+    include:
+      - master
+      - azure-pipelinestest
```
Comment (ttstanley) [inline azure-pipelines.yml:10]: I will change this branch trigger in a later PR. I wan't to make sure that there is smooth integration. So I don't want to take off this branch till we can validate everything works. Then I will put in another PR to get rid of this trigger.

File: nuget.config [hunk 2]
```diff
@@ -2,9 +2,20 @@
...
   </config>
   <packageSources>
+    <clear />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="vs-devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
   </packageSources>
-</configuration>
+  <trustedSigners>
+    <repository name="nuget" serviceIndex="https://api.nuget.org/v3/index.json">
```
Comment (adrianvmsft) [inline nuget.config:13]: This section should be limited to only packages referenced by this repo

File: nuget.config [hunk 3]
```diff
@@ -2,9 +2,20 @@
 <configuration>
   <config>
     <add key="repositorypath" value="packages" />
+    <add key="signatureValidationMode" value="require" />
   </config>
   <packageSources>
+    <clear />
     <add key="api.nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
-    <add key="vs-devcore" value="https://www.myget.org/F/vs-devcore/api/v3/index.json" />
+    <add key="vs-impl" value="https://pkgs.dev.azure.com/azure-public/vside/_packaging/vs-impl/nuget/v3/index.json" />
```
Comment (adrianvmsft) [inline nuget.config:10]: I wonder if we could remove this package source? Could we use only nuget?

File: azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+
+- task: PublishBuildArtifacts@1
+  displayName: Publish packages
+  inputs:
+    PathtoPublish: $(Build.ArtifactStagingDirectory)/packages
+    ArtifactName: packages
+    ArtifactType: Container
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
+
+- task: NuGetCommand@2
```
Comment (adrianvmsft) [inline azure-pipelines.yml:175]: I don't think this is needed (publishing to VS feeds). Could you check if those feeds contain older versions of the packages?
On the other hand, I don't think that causes any side effects

**Review by adrianvmsft — APPROVED** [review]

File: azure-pipelines.yml (hunk 4 above)
Comment (ttstanley) [reply azure-pipelines.yml:175]: from what I can tell the feed already has some versions out from 2017.
image

File: nuget.config (hunk 3 above)
Comment (ttstanley) [reply nuget.config:10]: we have to use vs-impl for microbuild components

**Pull request merged**